src/components/NumberInput/NumberInput.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import inputIdGenerator from '../../utils/inputIdGenerator';
+
+import './NumberInput.css';
+
+const NumberInput = ({ id = inputIdGenerator.nextIndex, labelText, value, onChange }) => (
+	<div className="NumberInput">
+		<label htmlFor={id}>
+			{labelText}
+		</label>
+		<input
+			id={id}
+			type="number"
```

Comment:
Double quotes detected

Reply:
Hrm.. does the linter not pick this up?

Reply:
Can you rebase onto master and re-lint

Reply:
Rebased locally.
`npm run lint` returns nothing in the CLI. 

Fixing anyway, but the linter issue will need looking into.

Reply:
master updated sorry, PR wasn't merged in yet - please rebase again

Reply:
That's cool - I haven't pushed fixes yet. 😄